File: Leetcode/Easy/containsDuplicate.js
const containsDuplicate = function(nums) {
    hash = {}
    for (let num of nums){
        if (num in hash){
            return true;
            break;
        } else {
            hash[num] = true;
        }
    }
    return false;
};

const nums = [3,4,5,2,3,1,4]
console.log(containsDuplicate(nums))

File: Leetcode/Easy/containsDuplicate.py
def containsDuplicate(nums):
    """
    :type nums: List[int]
    :rtype: bool
    """
    hash = {}
    for num in nums:
        if num in hash:
            return True
            break
        else:
            hash[num] = True
    return False

nums = [3,4,5,2,3,1,4]
print(containsDuplicate(nums))